src/remote_gym/remote_environment.py

```diff
@@ -68,6 +71,7 @@ def __init__(
         Args:
             url: URL to the machine where the remotely running environment application is hosted on.
             port: Open port on the remote machine (for communication with the remotely running environment application).
+            remote_args: Additional arguments sent to the remote environment
```

Comment:
Possible "hard-coded" fields should be documented here (e.g., `repo`).

Reply:
I moved this to a separate typed dict as it occurs on several places.